tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -3189,29 +3196,37 @@ node_modules/chrome-devtools-frontend/front_end/bindings/BlackboxManager.js(375,
 node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(237,46): error TS2339: Property 'valuesArray' does not exist on type 'Map<number, Breakpoint[]>'.
 node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(300,73): error TS2339: Property 'valuesArray' does not exist on type 'Map<number, Map<number, Breakpoint[]>>'.
-node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(314,58): error TS2339: Property 'keysArray' does not exist on type 'Map<UISourceCode, Map<number, Map<number, Breakpoint[]>>>'.
+node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(314,58): error TS2339: Property 'keysArray' does not exist on type 'Map<typeof UISourceCode, Map<number, Map<number, Breakpoint[]>>>'.
 node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(326,73): error TS2339: Property 'keysArray' does not exist on type 'Map<number, Map<number, Breakpoint[]>>'.
 node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(330,43): error TS2339: Property 'keysArray' does not exist on type 'Map<number, Breakpoint[]>'.
-node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(350,58): error TS2339: Property 'keysArray' does not exist on type 'Map<UISourceCode, Map<number, Map<number, Breakpoint[]>>>'.
+node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(337,41): error TS2339: Property 'uiLocation' does not exist on type 'typeof UISourceCode'.
+node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(350,58): error TS2339: Property 'keysArray' does not exist on type 'Map<typeof UISourceCode, Map<number, Map<number, Breakpoint[]>>>'.
 node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(396,17): error TS2339: Property 'remove' does not exist on type 'Breakpoint[]'.
 node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(399,34): error TS2339: Property 'delete' does not exist on type 'Multimap'.
 node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(442,23): error TS2339: Property 'remove' does not exist on type 'Breakpoint[]'.
 node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(444,23): error TS2339: Property 'remove' does not exist on type 'Map<number, Breakpoint[]>'.
 node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(446,19): error TS2339: Property 'remove' does not exist on type 'Map<number, Map<number, Breakpoint[]>>'.
-node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(448,40): error TS2339: Property 'remove' does not exist on type 'Map<UISourceCode, Map<number, Map<number, Breakpoint[]>>>'.
+node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(448,40): error TS2339: Property 'remove' does not exist on type 'Map<typeof UISourceCode, Map<number, Map<number, Breakpoint[]>>>'.
```

Comment:
looks like references to this type now get the static side instead of the instance side.